mobile/src/main/java/com/truckmuncher/truckmuncher/dagger/NetworkModule.java

```diff
@@ -107,4 +110,10 @@ public AuthService provideAuthService(RestAdapter.Builder builder, Account accou
                 .setRequestInterceptor(new AuthRequestInterceptor(appContext, account));
         return builder.build().create(AuthService.class);
     }
+
+    @Singleton
+    @Provides
+    public SearchService profideSearchService(RestAdapter adapter) {
```

Comment:
I think this should go in the `release` build type, not main. Since the other one is in `debug`